### PR TITLE
[aws-ints] fix cloudtrail regex

### PR DESCRIPTION
**Pull request by jvanbrie**
### What does this PR do?
Small modification to the arn for identifying cloudtrail logs for gov accounts.

### CODE REVIEW

**Review by ashwinjv — APPROVED** [review]

File: aws/logs_monitoring/parsing.py [hunk 1]
```diff
@@ -57,7 +57,7 @@
 rds_regex = re.compile("/aws/rds/(instance|cluster)/(?P<host>[^/]+)/(?P<name>[^/]+)")
 
 cloudtrail_regex = re.compile(
-    "\d+_CloudTrail_\w{2}-\w{4,9}-\d_\d{8}T\d{4}Z.+.json.gz$", re.I
+    "\d+_CloudTrail_\w{2}(|-gov)-\w{4,9}-\d_\d{8}T\d{4}Z.+.json.gz$", re.I
```
Comment (ashwinjv) [inline aws/logs_monitoring/parsing.py]: Do we need to do `-cn` as well?

Comment (dwikle) [inline aws/logs_monitoring/parsing.py]: are there any unit tests for this?

Comment (jvanbrie) [reply aws/logs_monitoring/parsing.py]: https://github.com/DataDog/datadog-serverless-functions/blob/master/aws/logs_monitoring/tests/test_parsing.py#L43
- Let me add some for these new cloudtrail types.

Comment (jvanbrie) [reply aws/logs_monitoring/parsing.py]: Yep, added. Also added support for cloudtrail digest.

**Review by dwikle — APPROVED** [review]

**Pull request merged**